.github/workflows/push-docker-develop.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: hermeznetwork/bridge-ui:1.5-fetestnet
+          tags: hermeznetwork/zkevm-bridge-ui:fetestnet
```

Comment:
The environment `fetestnet` was shut down by the ops team so I don't think we need to push this image anymore.

Reply:
Wasn't aware of that, I've just removed it